dev/pages/Toggle.vue

```diff
@@ -51,8 +51,9 @@ watch(() => variants.active, changeToggleModel)
     </token>
     <div>
     <token :state="[variants, toggleModel]">
-      <w-toggle radio-button :equal-width="isJustified" v-model="multiToggleModel" label="A very toggly label" :toggles="toggles" />
-      <w-button class="mt-16" small utility @click="isJustified = !isJustified">{{ isJustified ? 'Unjustify' : 'Justify' }}</w-button>
+      <w-button aria-controls="radio-button-group" class="mt-16" small utility @click="isJustified = !isJustified">{{ isJustified ? 'Unjustify radio button group' : 'Justify radio button group' }}</w-button>
```

Comment:
small thing, but the `A very toggly SMALL label` and the `A very toggly label` have the same id, so when trying to click on the small radio-button it first triggers the the bigger radio-button: 
<img width="288" alt="Skärmavbild 2024-03-06 kl  13 48 44" src="https://github.com/warp-ds/vue/assets/144792260/a45eee02-cb3c-4335-8fa6-8ccf24c8687f">

Reply:
Good catch! It was also the model that needed to be differentiated. Fixed in [bd6128a](https://github.com/warp-ds/vue/pull/142/commits/bd6128a50e060228b189337858e74768701c9778)